KerbalStuff/blueprints/profile.py

```diff
@@ -27,10 +28,16 @@ def view_profile(username: str) -> str:
                 abort(403)
     match = FORUM_PROFILE_URL_PATTERN.match(user.forumUsername)
     forum_url_username = match.groups()[1] if match else None
-    mods_created = sorted(user.mods, key=lambda mod: mod.created, reverse=True)
     # Remove unpublished mods from the list if it's not the accessing's user's own profile, or it is and admin.
-    if not current_user or (current_user.id != user.id and not current_user.admin):
-        mods_created = [mod for mod in mods_created if mod.published]
+    show_unpublished = current_user and (current_user.id == user.id or current_user.admin)
+    mods_created = list(map(lambda grp: (grp[0], sorted(grp[1],
```

Comment:
How about a small comment what all those grouping and sorting lambdas do?
```python
    # Group mods from this profile's user by game, and sort by creation date within each group
```

Reply:
Done.

Reply:
Thanks!